systems-management/nisysmgmt.yml

```diff
@@ -0,0 +1,2092 @@
+        default: 0
+        minimum: 0
+        example: 100
+      take:
+        description: The number of resources to return. The maximum value is 1000.
+        type: integer
+        format: int64
+        minimum: 0
+        maximum: 1000
+        example: 100
+      filter:
+        type: string
+        example: connected.data.state==CONNECTED
+        description: The criteria used to filter resources.
+      projection:
+        type: string
+        example: 'new(id,grains,lastUpdatedTimestamp)'
+        description: Specifies the projection for resources. This field can be used if we don't want to receive the entire model but only certain properties of the model.  
+      orderBy:
+        type: string
+        example: 'lastUpdatedTimestamp descending'
+        description: The order in which data returns.
+
+  QueryResponse:
+    description: Response of a query request.
+    title: Query Response
+    properties:
+      data:
+        description: The data returned by query.
+        type: array
+        items:
+          type: object
+        example:
+          - jid: '20190211135234448026_28320'
+      count:
+        description: The total number of resources that matched the query.
+        type: integer
+        format: int64
+        example: 1
+    required: [data, count]
+
+paths:
+  /:
+    get:
+      operationId: HandleGetVersionInfo
+      x-ni-request-all-privileges: true
+      summary: API information
+      tags: [versioning]
+      security: []
+      description: Gets permissions and operations for the Systems Management Service.
+      responses:
+        200:
+          description: OK.
+          schema:
+            description: Version information.
+            title: RootEndpointResponse
+            type: object
+            properties:
+              v1:
+                $ref: '#/definitions/V1Operations'
+              version:
+                description: The implementation version of the web service.
+                type: string
+        default:
+            $ref: '#/responses/Error'
+  /{version}:
+    get:
+      operationId: RootEndpointWithVersion
+      x-ni-request-all-privileges: true
+      summary: API version information
+      description: Returns available operations for a single version of the API.
+      tags: [versioning]
+      security: []
+      parameters:
+        - in: path
+          name: version
+          description: The version of the API to retrieve operations.
+          x-example: v1
+          type: string
+          required: true
+      responses:
+        200:
+          description: OK.
+          schema:
+            $ref: '#/definitions/V1Operations'
+        404:
+          description: Not Found.
+        default:
+            $ref: '#/responses/Error'
+  /v1/jobs:
+    post:
+      operationId: CreateJob
+      x-ni-privilege: Write
+      x-ni-operation: createOrCancelJobs
+      tags: [jobs]
+      summary: Create a job
+      description: Create a job.
```

Comment:
Would it make sense to also mention what is the effect of the route (creates a job and returns the newly created job)?